File: simple-stack-example-fragments/src/main/java/com/zhuinden/simplestackdemoexamplefragments/application/Injector.java
package com.zhuinden.simplestackdemoexamplefragments.application;

import com.zhuinden.simplestackdemoexamplefragments.application.injection.SingletonComponent;

/**
 * Created by Owner on 2017. 02. 03..
 */

public class Injector {
    public static SingletonComponent get() {
        return CustomApplication.get().getComponent();
    }
}

File: simple-stack-example-mvp-fragments/src/main/java/com/zhuinden/simplestackdemoexamplefragments/application/Injector.kt
package com.zhuinden.simplestackdemoexamplefragments.application

import com.zhuinden.simplestackdemoexamplefragments.application.injection.SingletonComponent

/**
 * Created by Zhuinden on 2018. 08. 19..
 */
object Injector {
    @JvmStatic
    fun get(): SingletonComponent = CustomApplication.getComponent()
}

